src/main.rs

```diff
@@ -619,14 +614,20 @@ fn main() {
                                         &script_loader,
                                         &map,
                                         character_information,
-                                        player_position,
+                                        Vector2::new(0, 0),
```

Comment:
Minor comment, maybe use Vector2::zero()

Reply:
Didn't know it existed, thanks!👍